lib/vmdb/loggers.rb

```diff
@@ -166,4 +100,5 @@ def self.apply_config_value(config, logger, key)
   end
 end
 
+require_relative "loggers/instrument"
```

Comment:
Why isn't this covered by the require in the next line @Fryguy ?

Reply:
@carbonin It's an ordering thing.  Some of the loggers require 'instrument' to be defined.  The alternate way to do this is to do `require_relative 'instrument'` in each logger that requires it...I had that originally, and switched to this, but I have no preference.